src/thor/route_action.cc

```diff
@@ -381,26 +381,28 @@ void thor_worker_t::path_arrive_by(Api& api, const std::string& costing) {
     if (temp_paths.empty())
       return false;
     for (auto& temp_path : temp_paths) {
-      // back propagate time information
-      if (!destination->date_time().empty() &&
```

Comment:
because this is arrive_by we are certain there will always be a date_time on the destination?

Reply:
ah i see you have it for the regular locations just not the first one